tests/generic_config_updater/test_bgp_prefix.py

```diff
@@ -22,6 +22,20 @@
 PREFIXES_V6_RE = r"ipv6 prefix-list PL_ALLOW_LIST_DEPLOYMENT_ID_0_COMMUNITY_{}_V6 seq \d+ permit {}"
 
 
+@pytest.fixture(autouse=True)
+def _ignore_allow_list_errlogs(duthosts, rand_one_dut_hostname, loganalyzer):
+    """Ignore expected failures logs during test execution."""
+    if loganalyzer:
+        IgnoreRegex = [
+            ".*ERR bgp#bgpcfgd: BGPAllowListMgr::Default action community value is not found.*",
```

Comment:
test_bgp_prefix_tc1_suite is to add the prefix in the allow list, it should not care if we have the allow list by default or not. Could you indicate why we still need to ignore this error?

Reply:
Testcase didn't care about allow list. It is the BGPAllowListMgr script in the image will check the allow list, which will cause the syslog error, then captured by log analyzer.